src/dom/getpositionedancestor.js

```diff
@@ -12,7 +12,7 @@ import global from './global';
 /**
  * For a given element, returns the nearest ancestor element which CSS position is not "static".
  *
- * @param {HTMLElement} element Native DOM element to be checked.
+ * @param {HTMLElement} [element] Native DOM element to be checked.
```

Comment:
What would this function be doing if the element was not specified? This param is required for it do work.

Reply:
It returns `null`. There is a [test](https://github.com/ckeditor/ckeditor5-utils/blob/7e6bd6631f8bcf677311736e7de095dd95327261/tests/dom/getpositionedancestor.js#L19-L21) for it. That said, _required_ make more sense indeed.